backend/internal/auth/mock.go

```diff
@@ -0,0 +1,16 @@
+package auth
+
+import (
+	"context"
+	"strings"
+)
+
+type MockAuthenticator struct{}
+
+func (a MockAuthenticator) VerifyToken(ctx context.Context, token string) (string, bool) {
```

Comment:
Document this with the structure of test tokens (that they are of the form `test-token<user ID>`)